assets/js/googlesitekit/modules/index.js

```diff
@@ -30,12 +30,54 @@ import './datastore';
 
 const Modules = {
 	createModuleStore,
+	/**
+	 * Activates a module on the server.
+	 *
+	 * Activate a module (based on the slug provided).
+	 *
+	 * @since 1.8.0
+	 *
+	 * @param {any}    args      Module action arguments.
+	 * @param {string} args.slug Slug of the module to activate.
+	 * @return {Promise} The dispatched action.
+	 */
 	activateModule: ( ...args ) => {
 		return Data.dispatch( STORE_NAME ).activateModule( ...args );
 	},
+	/**
+	 * Deactivates a module on the server.
+	 *
+	 * Deactivate a module (based on the slug provided).
+	 *
+	 * @since 1.8.0
+	 *
+	 * @param {any}    args      Module action arguments.
+	 * @param {string} args.slug Slug of the module to activate.
+	 * @return {Promise} The dispatched action.
+	 */
 	deactivateModule: ( ...args ) => {
 		return Data.dispatch( STORE_NAME ).deactivateModule( ...args );
 	},
+	/**
+	 * Registers a module.
+	 *
+	 * @since 1.13.0
+	 * @since 1.20.0 Introduced the ability to register settings and setup components.
+	 * @since 1.22.0 Introduced the ability to add a checkRequirements function.
+	 *
+	 * @param {Object}      args                         Module action arguments.
+	 * @param {string}      args.slug                    Module slug.
+	 * @param {string}      [args.name]                  Optional. Module name. Default is the slug.
+	 * @param {string}      [args.description]           Optional. Module description. Default empty string.
+	 * @param {string}      [args.icon]                  Optional. Module icon. Default empty string.
+	 * @param {number}      [args.order]                 Optional. Numeric indicator for module order. Default 10.
+	 * @param {string}      [args.homepage]              Optional. Module homepage URL. Default empty string.
+	 * @param {WPComponent} [args.settingsEditComponent] Optional. React component to render the settings edit panel. Default none.
+	 * @param {WPComponent} [args.settingsViewComponent] Optional. React component to render the settings view panel. Default none.
+	 * @param {WPComponent} [args.setupComponent]        Optional. React component to render the setup panel. Default none.
```

Comment:
Sorry @benbowler but it looks like these were recently updated after #2413 being merged 😄 
https://github.com/google/site-kit-wp/blob/80c932da36bc35fae7637015e4ba31099979d543/assets/js/googlesitekit/modules/datastore/modules.js#L230-L235

Also, note that these are named `settings` in the action docs. I understand the way the args are named below forces us to use the same name here but it's actually kind of cheating – the real `args` below is an _array_ of all arguments passed to the function but we're documenting it above as an object. I would suggest we replace the `args` spreading with the real args instead but let's wait and see what @felixarntz says about that.

Reply:
Updated, we'll need to make sure these keep in sync when the actions are updated in future.